src/service-worker.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-undef, no-restricted-globals */
```

Comment:
You might be able to replace this line with `/* global self, workbox */`

Reply:
eslint isn't into it:
"the new `no-restricted-globals` rule disallows the use of self, even if the top of the file includes `/* global self */`."
https://github.com/airbnb/javascript/issues/1632